sdk/src/main/java/com/oursky/authgear/AuthgearCore.kt

```diff
@@ -124,6 +125,15 @@ internal class AuthgearCore(
             return true
         }
 
+        fun handleAuthgearDeepLink(deepLink: String): Boolean {
```

Comment:
A second thought, I think we need a new method for this feature. I am going use JS Promise to explain.

`open()`: resolve if canceled, reject otherwise
`authenticate()`: resolve if got authorization code, reject otherwise (including cancel)
the new method: resolve if `authgearsdk://host/path` is detected, reject otherwise (including cancel)

This allows the developer to know whether the end-user actually finished changing their password. If we keep using `open()`, we cannot convey this information.